### ISSUE TITLE
128MB Xiaofang?

### ISSUE THREAD

**Issue by Swiftnesses** [Dafang-Hacks/uboot#2]
My device originally said it was 128MB during startup - I had to flash the uboot from the original folder in order to get hacks to work (it now reports 64MB). Is it possible to get a version of the bootloader that can use the entire 128MB my camera appears to have available?

**Comment by Swiftnesses** [comment]
I flashed the 128MB V2, all seems to be working...

**Comment by EliasKotlyar** [comment]
ok :) so dont flash the wrong uboot :)

**Comment by Swiftnesses** [comment]
Hey @EliasKotlyar,

I've tried most of the bootloaders, inc the plain defang ones and the xaiofang1s ones (v1 didn't detect the sd and required pins set like Paul had some time ago, v2 boots, albeit with 64mb only).

I have 2 issues it seems:

1) I require a 128MB version (v2?, see issue 2 hoever!)...
2) None of the bootloaders with rootfs allow me to connect to wifi (made the edits to /etc) no luck.

Can you help, I'm 2 days into this, no solution yet, but so close!

**Comment by Swiftnesses** [comment]
Log here: https://pastebin.com/athmEyZG

If I can help with commands etc, let me know :)

**Comment by Swiftnesses** [comment]
@EliasKotlyar No wifi fixed by uncommenting the line in /etc/init.d/rcS.

Now, if I could just use the entire 128MB and enable 1080P, I'd be super happy :)

**Comment by parvata** [comment]
Hi, Can you tell me if there is a place where I can buy just the motherboard for the dafang device? I may have wrecked the soldering contacts on the board while de-soldering the chip. Thanks for any info.

**Comment by EliasKotlyar** [comment]
Unfortunately no - i really dont know where to buy spare parts. However you can buy a newer one, they are pretty cheap.

**Comment by parvata** [comment]
yeah, that is what I have done :-). But it would be nice if there was a possibility. oh well...

**Comment by DavidFFerreira** [comment]
and for T20L what to use?
